kafka/ed-fi-kafka-connect-transforms/src/main/java/org/edfi/kafka/connect/transforms/GenerateIndexFromDmsResource.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2019 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ *  SPDX-License-Identifier: Apache-2.0
+ *  Licensed to the Ed-Fi Alliance under one or more agreements.
+ *  The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ *  See the LICENSE and NOTICES files in the project root for more information.
+ */
```

Comment:
The Apache license states: "You must cause any modified files to carry prominent notices stating that You changed the files;". 

Our license header partially duplicates the Apache license information above that. Let's replace our header with this:
```java
/*
 * SPDX-License-Identifier: Apache-2.0
 * This file has been modified by the Ed-Fi Alliance.
 */
```

We can make that same license header change in any other `.java` files.